src/widgets/radio.jsx

```diff
@@ -19,6 +19,146 @@ var captureScratchpadTouchStart =
 
 var cx = React.addons.classSet;
 
+var Choice = React.createClass({
+    propTypes: {
+        checked: React.PropTypes.bool,
+        classSet: React.PropTypes.shape,
+        clue: React.PropTypes.shape,
+        correct: React.PropTypes.bool,
+        content: React.PropTypes.shape,
+        disabled: React.PropTypes.bool,
+        groupName: React.PropTypes.string,
+        onChecked: React.PropTypes.func,
+        type: React.PropTypes.string
+    },
+
+    getDefaultProps: function() {
+        return {
+            checked: false,
+            classSet: {},
+            correct: false,
+            disabled: false,
+            type: 'radio'
+        };
+    },
+
+    render: function() {
+        this.props.classSet[ApiClassNames.RADIO.OPTION] = true;
+        this.props.classSet[ApiClassNames.RADIO.SELECTED] = this.props.checked;
+
+        return <div className={cx(this.props.classSet)}>
+            <span className="checkbox">
+                <input
+                    type={this.props.type}
+                    name={this.props.groupName}
+                    checked={this.props.checked}
+                    disabled={this.props.disabled}
+                    onClick={(e) => {
+                        // Avoid sending this to the parent
+                        e.stopPropagation();
+                    }}
+                    onChange={(e) => {
+                        this.props.onChecked(e.target.checked);
+                    }} />
+            </span>
+            {/* A pseudo-label. <label> is slightly broken on iOS,
+                so this works around that. Unfortunately, it is
+                simplest to just work around that everywhere. */}
+            <span className={
+                    ApiClassNames.RADIO.OPTION_CONTENT + " " +
+                    ApiClassNames.INTERACTIVE
+                }
+                style={{ cursor: "default" }}>
+                <div>
+                    {this.props.content}
+                </div>
+            </span>
+            {Exercises.cluesEnabled && this.props.checked &&
+                <div className="perseus-radio-clue">
+                    {this.props.clue}
+                </div>}
+        </div>
+    }
+});
+
+var ChoiceNoneAbove = React.createClass({
+    getDefaultProps: function() {
+        return {
+            correct: true
+        }
+    },
+
+    render: function() {
+        if (!this.props.correct) {
```

Comment:
I support naming it `showContent` or something! I think that plus a comment (a good comment of why we're doing something is valuable even in the best code) would make this a lot clearer :).